package.json

```diff
@@ -3,8 +3,11 @@
   "version": "8.0.0",
   "description": "npm's markdown parser",
   "main": "index.js",
+  "browser": "dist/marky-markdown.js",
   "scripts": {
-    "test": "standard-format -w && standard && mocha --timeout 5000"
+    "build": "browserify index.js -i highlights -s markyMarkdown > dist/marky-markdown.js",
+    "test": "standard-format -w && standard && mocha --timeout 5000",
+    "prepublish": "build"
```

Comment:
Travis is complaining that it can't find the `build` command. `npm run build` here?